src/routes/(iframed)/+page.svelte

```diff
@@ -49,6 +50,22 @@
   function onResize(width: number, height: number) {
     sendResizeMessage(width, height);
   }
+
+  let settings = getDefaultSettings();
+  let settingError = true;
+
+  onMount(() => {
+    const url = new URL(window.location.href);
+    settings = loadSettingsWithURLOverrides(url);
+
+    setTheme(settings.theme);
+
+    if (settings.game.length > 0 && settings.engine.length > 0) {
+      settingError = false;
+      playbackState.load(fetch, settings);
+      initWindowMessages();
+    }
```

Comment:
This is basically what was in `load` before

Moved it to `onMount` to quarantee it runs only in the browser, so we can use `window.location.href` to get the current url, with `searchParams`!